.github/workflows/release.yml

```diff
@@ -18,6 +18,36 @@ jobs:
       - name: Run tests
         run: make unit-tests
 
+  docs:
```

Comment:
Perhaps, we could put the docs in a separate workflow?
It wouldn't hooked to the release workflow in that case.
Not a big change in any case.